assets/app/view/create_game.rb

```diff
@@ -113,6 +157,7 @@ def mode_input(mode, text)
         elm = Native(@inputs[:max_players]).elm
         elm.value = [elm.value.to_i, elm.min.to_i].max
         store(:num_players, elm.value.to_i)
+        update_optional_rules_store
```

Comment:
should we have one method that sets all the stuff, like num players and optional rules?

Reply:
`update_inputs` does this now